website/yarn.lock

```diff
@@ -4170,6 +4196,11 @@ follow-redirects@^1.0.0, follow-redirects@^1.14.7:
   resolved "https://registry.yarnpkg.com/follow-redirects/-/follow-redirects-1.15.5.tgz#54d4d6d062c0fa7d9d17feb008461550e3ba8020"
   integrity sha512-vSFWUON1B+yAw1VN4xMfxgn5fTUiaOzAJCKBwIIgT/+7CuGy9+r+5gITvP62j3RmaD5Ph65UaERdOSRGUzZtgw==
 
+follow-redirects@^1.15.0:
```

Comment:
What are these changes for?

Reply:
These are automatically generated by docusaurus and yarn. I was googling if we could just ignore this file, but apparently it need to be checked in =/